src/Nethermind/Nethermind.Synchronization/Metrics.cs

```diff
@@ -73,5 +75,30 @@ public static class Metrics
         [GaugeMetric]
         [Description("Sync time in seconds")]
         public static long SyncTime;
+
+        [DetailedMetric]
+        [Description("Snap range result")]
+        [KeyIsLabel("is_storage", "result")]
+        public static NonBlocking.ConcurrentDictionary<SnapRangeResult, long> SnapRangeResult { get; set; } = new();
```

Comment:
are you positive `NonBlocking` is better than normal? Our tests shown that in not very-heavy cases it is actually worse.